Dockerfile

```diff
@@ -1,7 +1,7 @@
-FROM jinaai/jina:2.0.0rc7
+FROM jinaai/jina:2.0.0rc8
```

Comment:
needs to be rc10 to be able to build the container

Reply:
@davidbp it's already rc10, maybe refresh?